docs/blog/2018-10-06-deploying-gatsby-to-azure/index.md

```diff
@@ -133,22 +135,41 @@ This will output a production build of your blog into the `public` folder.
 
 The easiest way to deploy your blog to your storage account is to use the `Azure Storage Extension` within VS Code.
 
-[Install the Azure Storage Extension](vscode:extension/ms-azuretools.vscode-azurestorage)
+[Install the Azure Storage Extension - VS Code](vscode:extension/ms-azuretools.vscode-azurestorage)
```

Comment:
I think the `vscode:`-dependent link is probably not a great experience for people who aren't familiar with VS Code. It'll seem like a dead link, which isn't a great experience. (I'm also not confident that these types of links are reliable cross-browser/platform.) The VS Code marketplace supports jumping into VS Code and installing directly, so I'd push for just linking to that to avoid any potential confusion for readers.

Reply:
Yeah good point. I've removed to ensure it's as generic as possible.